CONTRIBUTING.md

```diff
@@ -141,6 +141,33 @@ to email the patch to <gap@gap-system.org>.  You can refer to
 [this tutorial](https://burzalodowa.wordpress.com/2013/10/05/how-to-send-patches-with-git-send-email/)
 on how to do this.
 
+## Reviewing code contributions
+
+When reviewing code, you can use the following list as a checklist.
```

Comment:
This text seems to know that readers already know what code reviews are (and what their ultimate purpose is), and how to do them. Might be worthwhile to elaborate a bit on that, and/or link to source explaining it. For the latter (how to), https://help.github.com/articles/about-pull-request-reviews/ might be a good URL.